src/bin.js

```diff
@@ -1,3 +1,6 @@
 import postNpmInstall from './index.js';
 
-postNpmInstall();
+const args = process.argv.slice(2);
+setToCi = args.length > 0 && args[0] === 'ci';
```

Comment:
1. Should we include a `const` here?
2. Would it be just as readable to directly access `process.argv`?

```suggestion
const setToCi = process.argv.length > 2 && process.argv[2] === 'ci';
```